src/appEntry.js

```diff
@@ -0,0 +1,2 @@
+import 'bootstrap';
```

Comment:
bootstrap is imported here but never used.

Reply:
Imported for webpack

Reply:
I wouldn't worry too much about unused imports at this point. We can easily clean them up later.